src/components/DuiAndCrimHis.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Radio, Form, Input, Button } from 'antd';
+
+const validateMessages = {
+  // eslint-disable-next-line no-template-curly-in-string
+  required: '${label} is required!',
+};
+
+const DuiAndCrimHis = () => {
+  const [componentSize, setComponentSize] = React.useState('default');
+  const [requiredMark, setRequiredMarkType] = React.useState('optional');
```

Comment:
This is causing a page break when check boxes are deselected or when all values are removed from “vehicle type”. Please use only the "required" attribute in the respective Form items. You can use a simple html tag with some styling to create the gray "(optional)" text.

Reply:
Another quick note about this, `useState` should be imported from React:
```js
import React, { useState } from 'react';
...
const [componentSize, setComponentSize] = useState('default');
const [requiredMark, setRequiredMarkType] = useState('optional');
```